File: src/main/java/com/baulsupp/oksocial/services/linkedin/LinkedinUtil.java
package com.baulsupp.oksocial.services.linkedin;

import com.google.common.collect.Sets;
import java.util.Collections;
import java.util.Set;
import okhttp3.Request;

public class LinkedinUtil {
  private LinkedinUtil() {
  }

  public static final Set<String> API_HOSTS =
      Collections.unmodifiableSet(Sets.newHashSet(
          "api.linkedin.com")
      );

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.linkedin.com" + s).build();
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/linkedin/LinkedinUtil.kt
package com.baulsupp.oksocial.services.linkedin

import okhttp3.Request

object LinkedinUtil {

    val API_HOSTS = setOf((
            "api.linkedin.com"))

    fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
        return requestBuilder.url("https://api.linkedin.com" + s).build()
    }
}
